int8 cord_x 
int8 cord_y 
---
bool status 
